packages/cactus-plugin-keychain-google-sm/src/main/json/openapi.json

```diff
@@ -76,6 +76,57 @@
           }
         }
       }
+    },
+    "/api/v1/plugins/@hyperledger/cactus-plugin-keychain-google-sm/delete-keychain-entry": {
+      "post": {
+        "x-hyperledger-cactus": {
+          "http": {
+            "path": "/api/v1/plugins/@hyperledger/cactus-plugin-keychain-google-sm/delete-keychain-entry",
+            "verbLowerCase": "post"
+          }
+        },
+        "operationId": "deleteKeychainEntryV1",
+        "summary": "Deletes an entry under a key on the keychain backend.",
+        "parameters": [],
+        "requestBody": {
+          "$ref": "../../../../cactus-core-api/src/main/json/openapi.json#/components/requestBodies/keychain_delete_entry_request_body"
```

Comment:
@Leeyoungone Is it possible now to update these relative file paths to links pointing to a specific release's git tags instead? I can't remember of the parent PR was merged already or not and if it was, then was a new release issued since then to tag it or not?